### DISCUSSION TITLE
Community themes gallery

### DISCUSSION THREAD

**Discussion by rnd195** [marp-team/marp#285] [Ideas] [answered]
I am not sure whether this was ever brought up, but I think it would be interesting to have some sort of a gallery for community-submitted themes. For example, the Typora project has a dedicated repository where users can open pull requests to submit their own themes (see here https://github.com/typora/theme.typora.io and https://theme.typora.io/).

**Comment by yhatt** [comment, marked as answer]
Currently I'm busy with keeping maintain the whole of Marp ecosystem and I have no time to curate community themes and polish the gallery. But making a site of Marp theme gallery by our community is always welcome! I'll happy to introduce a gallery at the entrance repository. :smile:

**Reply by rnd195** [reply]
I understand, thanks!
